CondCore/L1TPlugins/src/UpgradeRecords2.cc

```diff
@@ -26,8 +22,6 @@
 #include "CondFormats/DataRecord/interface/L1TMuonGlobalParamsRcd.h"
 #include "CondFormats/DataRecord/interface/L1TMuonGlobalParamsO2ORcd.h"
 
-REGISTER_PLUGIN(L1TMuonEndcapParamsRcd,  L1TMuonEndCapParams); // Temporary copy to avoid crashes - AWB 28.08.17
```

Comment:
@rekovic removing this plugin creates the problem in the tests. In a standalone run of wf 136.85 I get:

----- Begin Fatal Exception 04-Oct-2018 17:04:53 CEST-----------------------
An exception of category 'MultiplePlugins' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing ESSource: class=PoolDBESSource label='GlobalTag'
Exception Message:
The plugin 'L1TMuonEndcapParamsRcd@NewProxy' is found in multiple files 
 '"plugin-poisoned-CondCoreL1TPlugins.so"'
 '"plugin-poisoned-CondCoreL1TPlugins.so"'
in directory '/build/fabiocos/103X/24659/CMSSW_10_3_X_2018-10-03-2300/lib/slc6_amd64_gcc700/poisoned'.
The code must be changed so the plugin only appears in one plugin file. You will need to remove the macro which registers the plugin so it only appears in one of these files.
  If none of these files register such a plugin, then the problem originates in a library to which all these files link.
The plugin registration must be removed from that library since plugins are not allowed in regular libraries.
----- End Fatal Exception -------------------------------------------------

Restoring back this file to its master version solves the issue, re-establishing the old plugin. Anyway @smuzaffar the message looks unclear to me, and I do not understand the reason for the difference wrt to the Jenkins output:

----- Begin Fatal Exception 01-Oct-2018 11:00:04 CEST-----------------------
An exception of category 'PluginLibraryLoadError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing ESSource: class=PoolDBESSource label='GlobalTag'
Exception Message:
unable to load /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_10_3_X_2018-09-30-2300/lib/slc6_amd64_gcc700/poisoned/plugin-poisoned-CondCoreL1TPlugins.so because /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_10_3_X_2018-09-30-2300/lib/slc6_amd64_gcc700/poisoned/plugin-poisoned-CondCoreL1TPlugins.so: cannot open shared object file: No such file or directory
----- End Fatal Exception -------------------------------------------------

Reply:
@fabiocos , I think one error is generated at the time of plugin registration and other at runtime but @Dr15Jones can tell more about these messages.
All these show that a plugin was removed while some one still try to access it.

Reply:
The output from Jenkins is what we are always supposed to generate. I have never seen the first one. The first one appears to have an incorrectly generated .edmplugincache file which includes multiple entries for plugin-poisoned-CondCoreL1TPlugins.so.